src/chrome-launcher.ts

```diff
@@ -373,28 +379,30 @@ class Launcher {
   }
 
   kill() {
-    return new Promise<void>((resolve, reject) => {
-      if (this.chrome) {
-        this.chrome.on('close', () => {
-          delete this.chrome;
+    return new Promise<void>((resolve) => {
+      if (this.chromeProc) {
+        this.chromeProc.on('close', () => {
+          delete this.chromeProc;
           this.destroyTmp().then(resolve);
         });
 
-        log.log('ChromeLauncher', `Killing Chrome instance ${this.chrome.pid}`);
+        log.log('ChromeLauncher', `Killing Chrome instance ${this.chromeProc.pid}`);
         try {
           if (isWindows) {
-            // While pipe is the default, stderr also gets printed to process.stderr
-            // if you don't explicitly set `stdio`
-            execSync(`taskkill /pid ${this.chrome.pid} /T /F`, {stdio: 'pipe'});
+            // https://github.com/GoogleChrome/chrome-launcher/issues/266
+            const taskkillProc = spawnSync(
+                `taskkill /pid ${this.chromeProc.pid} /T /F`, {shell: true, encoding: 'utf-8'});
+
+            const {stderr} = taskkillProc;
+            if (stderr) log.error('ChromeLauncher', `taskkill stderr`, stderr);
           } else {
-            if (this.chrome.pid) {
-              process.kill(-this.chrome.pid);
+            if (this.chromeProc.pid) {
+              process.kill(-this.chromeProc.pid, 'SIGKILL');
             }
           }
         } catch (err) {
           const message = `Chrome could not be killed ${err.message}`;
           log.warn('ChromeLauncher', message);
-          reject(new Error(message));
```

Comment:
The smoke tests still pass even with this rejection right?

Reply:
no they don't. 

removing this rejection is deliberate actually. as mentioned in #266, pptr and pw deliberately do not reject in this case. 

and i've been convinced that this is reasonable. there appears to be a race condition and taskkill is attempting to kill processes that are already gone. what's interesting is that when this happens. the processes that fail to be killed (cuz they're not found) are all child proc of the primary chrome proc.  the chrome proc is always successfully killed.

take a peek at the SO item linked in https://github.com/microsoft/playwright/pull/7500 .... PW used this `MEMUSAGE` hack for a while but then determined it wasnt needed. so i'm following their latest pattern.

i think it's still important we have some visibility on when taskkill returns an error, but it's not a fatal situation.